sim/src/generic_eps_hardware_model.cpp

```diff
@@ -61,19 +78,36 @@ namespace Nos3
         sim_logger->info("Generic_epsHardwareModel::Generic_epsHardwareModel:  Now on time bus named %s.", _command_bus_name.c_str());
 
         /* Initialize status for battery and bus */
-        std::string battv, battv_temp, solararray, solararray_temp;
+        std::string battv, battv_temp, solararray, solararray_temp, batt_watt_hrs, always_on_v, always_on_a;
+
+        // Below, the battery watt-hrs variable arbitrarily selected - it could well 
+        // do to be changed to be more in line with true spacecraft values.
+        // Additionally, the current values (as indicated) are placeholders and should
+        // probably be changed to something more correct.
+
+        _power_per_panel = 26.91; //Power generated, in Watts; data taken from GTOSat
```

Comment:
Can this as well as the bus voltages and currents be added to the simulator config input?
Appears to me that everything is working otherwise after test and review!